tests/api/api/test_runs.py

```diff
@@ -224,6 +224,54 @@ def test_list_runs_times_filters(db: Session, client: TestClient) -> None:
     )
 
 
+def test_list_completed_runs(db: Session, client: TestClient) -> None:
+    runs_endpoint = f"/api/projects/{config.default_project}/completed_runs"
+    base_time = datetime.now(timezone.utc)
+    total_runs = 10
+
+    runs = []
+    counter = 0
+    while counter < total_runs:
```

Comment:
While this does work, I'm wondering why to use a while here and maintaining a counter instead of `for counter in range(total_runs):` ?

Reply:
Good point! I replaced both `while` loops with `for` counterparts.